addons/prometheus/prometheus.yaml

```
@@ -9,18 +9,18 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.47.0-2"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.47.0-1"
```

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.47.0-3"
```
why did it decrement ?